lib/Email/Address.pm

```diff
@@ -37,25 +36,25 @@ for (1 .. $COMMENT_NEST_LEVEL) {
   $ccontent = qr/$ctext|$quoted_pair|$comment/;
   $comment  = qr/\($ccontent*\)/;
 }
-my $cfws           = qr/$comment|\s+/;
+my $cfws           = qr/(?>$comment|\s+)/;
```

Comment:
http://perldoc.perl.org/5.8.8/perlre.html#(%3f%3epattern) says it is supported but experimental. I haven't got perl 5.8 installed to test, I'm afraid; later commits in this PR are definitely 5.10 only, but I hoped this one would be okay so that 5.8 could get the space backtracking improvements, if not the comment backtracking.